src/Documentation/What's-new-in-Orleans.md

```diff
@@ -5,6 +5,70 @@ title: What's new in Orleans
 
 # What's new in Orleans?
 
+## [v1.4.1](https://github.com/dotnet/orleans/releases/tag/v1.4.1) March 27th 2017
+
+### Release notes
```

Comment:
should we continue duplicating the release notes in so many places? Can't we just link to the release or the changelog?